src/test/unit/utils/RandomTest.cpp

```diff
@@ -327,66 +312,75 @@ TEST(RandomTest, ReturnInCorrectRange) {
   }
 }
 
+
 TEST(RandomTest, getUInt64) {
   // tests for getUInt64
   Random r1(1);
-  ASSERT_EQ(3723745761376425000, r1.getUInt64())
+  ASSERT_EQ(2469588189546311528u, r1.getUInt64())
```

Comment:
all the values are changed, but that is ok for this one time (changing RNG engine)